cluster-autoscaler/processors/nodegroupset/aws_nodegroups.go

```diff
@@ -20,8 +20,10 @@ import (
 	schedulernodeinfo "k8s.io/kubernetes/pkg/scheduler/nodeinfo"
 )
 
-// IsAwsNodeInfoSimilar adds AWS specific node labels to the list of ignored labels.
-func IsAwsNodeInfoSimilar(n1, n2 *schedulernodeinfo.NodeInfo) bool {
+// CreateAwsNodeInfoComparator returns a comparator that checks if two nodes should be considered
+// part of the same NodeGroupSet. This is true if they match usual conditions checked by IsCloudProviderNodeInfoSimilar,
+// even if they have different AWS-specific labels.
+func CreateAwsNodeInfoComparator(ignoredLabels []string) NodeInfoComparator {
```

Comment:
Please rename parameter to "extraIgnoredLablels" to point that this are not the only one ignored (there as still `Basic` ones).

Same for other comparator implementations.